src/helpers/review.js

```diff
@@ -0,0 +1,30 @@
+
+import fetch from "node-fetch";
+
+const API = "https://plankton-app-xhkom.ondigitalocean.app/api/reviews";
+
+export async function sendReview(movieId, data) {
+
+  const body = {
+    "data": {
+      comment: data.comment,
+      rating: data.rating,
+      author: data.author,
+      verified: false,
+      movie: movieId,
+      createdAt: new Date().toISOString(),
```

Comment:
Do you need to set this manually? I would expect that it's assigned by the CMS automatically.